src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double rightLeg;
+    private double base;
+
+    public RightTriangle(double rightLeg, double base, String color) {
+        super(color);
+        this.rightLeg = rightLeg;
+        this.base = base;
+    }
+
+    public void setRightLeg(double rightLeg) {
+        this.rightLeg = rightLeg;
+    }
+
+    public void setBase(double base) {
+        this.base = base;
+    }
+
+    public double getRightLeg() {
+        return rightLeg;
+    }
+
+    public double getBase() {
+        return base;
+    }
+
+    @Override
+    public double area() {
+        return (base * rightLeg)/2;
```

Comment:
To improve readability, consider storing the result of the expression `(base * rightLeg) / 2` in a local variable before returning it. This makes it easier to understand what is being calculated and returned.